### DISCUSSION TITLE
CM10ZW - Orvibo shows only one endpoint

### DISCUSSION THREAD

**Discussion by bummzulu** [Koenkk/zigbee2mqtt#13357] [Devices]
Hi,

Having installed an CM10ZW that I wanted to use to read the status of dry contacts. I was able to pair it to my network but unfortunately it only shows one endpoint available (L1). L2 and L3 throw an error when I try to toggle them using the dashboard.

![image](https://user-images.githubusercontent.com/46400859/182188480-39d669f8-b311-49b1-8405-4abe2fc415a8.png)

Example:
Device '0x6904000029032288' has no endpoint 'l2'

Besides that the L1 toggle does not seem to trigger anything on the module as I dont hear a relay to be triggered - regardless of how the device is jumpered - but this is a side issue that I will investigate further once the device is setup correctly.

Some background information:
Zigbee2MQTT version 1.26.0 (commit #cd23aea)
zigbee-herdsman (0.14.40)
Coordinator firmware version: '{"meta":{"maintrel":1,"majorrel":2,"minorrel":7,"product":1,"revision":20210708,"transportrev":2},"type":"zStack3x0"}

Is anyone having a hint?

**Comment by jymorel** [comment]
Hi, I just installed it and have the same issue ... Did you solve it and how ? Thanks

**Comment by bummzulu** [comment]
Unfortunately not. No idea who could help here.

jymorel ***@***.***> schrieb am Mi. 7. Sep. 2022 um 18:27:

**Reply by jymorel** [reply]
Solved by my-self. I found this. 
You have to move to "on" switch 3 on the module and the 3 endpoints will be available after reinitializing the module et pairing again.
![orvibo](https://user-images.githubusercontent.com/96637729/188992528-143ff91b-15bb-4196-8642-8220ade67a88.png)

**Reply by bummzulu** [reply]
Thanks a lot for that! I was able to hear the relay being switched by turning on option 3 - but never thought about pairing the device again as the first relay was following the commands.

Great finding and thanks for sharing. Will give it a try.

**Reply by bummzulu** [reply]
Btw. What is your use-case for it? I am trying to read out the status of a dry contact with it.

**Comment by jymorel** [comment]
just on/off (0.5 sec) for opening / closing portal, gate and garage door
